examples/simple-app/index.html

```diff
@@ -0,0 +1,6 @@
+<html>
```

Comment:
no DOCTYPE 😱

Reply:
It's an example
![tenor](https://user-images.githubusercontent.com/46549/88816092-a7894900-d181-11ea-93a6-7da267961ef7.gif)